.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint:fix && git add .
```

Comment:
This wont add the fixes to the commit. is this intended?

Reply:
Oh. As I usually separate my commits, this just unintentionally add everything. It should only add what has been staged then